components/Navigation/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Link from "next/link";
+
+export default function Navigation() {
+  return (
+    <ul>
+      <li>
+        <Link href="/">Spotlight</Link>
+      </li>
+      <li>
+        <Link href="../../art-pieces">Art Pieces</Link>
```

Comment:
maybe we can try this:
```suggestion
        <Link href="/art-pieces">Art Pieces</Link>
```